landing-page/content/common/how-to-release.md

```diff
@@ -21,6 +21,18 @@ disableSidebar: true
  - limitations under the License.
  -->
 
+## Introduction
+
+This page walks you through the release process of the Iceberg project.
+
+Decisions about releases are made by three groups:
+
+* Release Manager: Does the work of creating the release, signing it, counting votes, announcing the release and so on. Doesn't have to be a committer but then should find a committer who helps out with some of the steps.
+* The community: Performs the discussion of whether it is the right time to create a release and what that release should contain. The community can also cast non-binding votes on the release.
```

Comment:
We should probably define what "votes on the release" means in this context. May be fine to just link to the Apache Docs 

https://www.apache.org/foundation/voting.html
Which has a lot more through information 

And https://www.apache.org/legal/release-policy.html 
as well

Reply:
Thanks for the observation! There is already a section that describes what voting is, also has a link to the Apache voting page. I added a link from this part of the doc to the voting section.
Additionally, I added a link for the apache release-policy page as well.